### PR TITLE
feat(alerts): Add new scopes for editing/viewing alert rules

### PR DESCRIPTION
**Pull request by davidenwang**
This PR adds alerts permission group with `alerts:read` and `alerts:write`. These scopes are added to all roles within the app except owners and also added to the relevant issue alert and metric alert endpoints. Eventually we will have a setting to turn on `alerts:write` for owners

There are also some frontend changes to enable buttons which were previously behind `projects:write`

### CODE REVIEW

File: src/sentry/incidents/endpoints/serializers.py [hunk 1]
```diff
@@ -291,8 +291,10 @@ class AlertRuleSerializer(CamelSnakeModelSerializer):
     environment = EnvironmentField(required=False, allow_null=True)
     # TODO: These might be slow for many projects, since it will query for each
     # individually. If we find this to be a problem then we can look into batching.
-    projects = serializers.ListField(child=ProjectField(), required=False)
-    excluded_projects = serializers.ListField(child=ProjectField(), required=False)
+    projects = serializers.ListField(child=ProjectField(scope="alerts:write"), required=False)
```
Comment (wedamija) [inline src/sentry/incidents/endpoints/serializers.py:294]: I think this should be `projects:read`, although probably not a huge deal.

**Review by wedamija — APPROVED** [review]
This lgtm, just need to fix tests. We should probably wait to merge until after we meet with adi as well.

**Review by scttcper — APPROVED** [review]
exciting

**Comment by davidenwang** [comment]
After discussing with @adhiraj the plan is to have a setting to turn on alert creation/editing for members. Following Dan's suggestion I'm gonna merge this in with members not having the `alert:write` permission and then add that in based on a setting in a follow-up PR

**Review by wedamija — APPROVED** [review]
Looks good!

**Comment by wedamija** [comment]
Actually, probably worth changing to title to reflect what this does - adds new scopes for editing/viewing alert rules.

**Pull request merged**